### PR TITLE
PR for Jenna

### PR DESCRIPTION
**Pull request by eshipunova**
I’m speaking with myself, number one, because I have a very good brain and I’ve said a lot of things... I know what I’m doing and I listen to a lot of people, I talk to a lot of people and at the appropriate time I’ll tell you who the people are. But I speak to a lot of people. I always told people, you know I'm a very smart guy. I got good marks. 
(citation needed)

### CODE REVIEW

File: my_main.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
+# I think book-reading should be an easy one. Or a math computation of some kind.
```
Comment (jennasilla) [inline my_main.py:2]: Remove comments unrelated to the code and its function.

File: my_main.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Inputs: number
+# Outputs: all possible ways of adding digits together, printed out
+
```
Comment (jennasilla) [inline my_main.py:5]: Describe the parameters, expected returns, and purpose of the function

File: my_main.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+def print_all(n):
+    s = str(n)
+    for d in s:
+        # I should make some tea. Black tea? To stay awake. I need to make good progress by tomorrow
```
Comment (jennasilla) [inline my_main.py:9]: Remove comments unrelated to the code

File: my_main.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        # 4 = 4, 2 + 1 = 3 =>
+        # 4 + 3 = 7
+        # 43|71730: 4 + 3 = 7, 7 + 1 + 7 + 3 + 0 = 18 => 7 = 7, 1 + 8 = 9 => 7 + 9 = 16 => 1 + 6 = 7
+        # 437|1730: 4 + 3 + 7 = 14, 1 + 7 + 3 + 0 = 11 => 1 + 4 = 5, 1 + 1 = 2 => 5 + 2 = 7
+        # 4371|730: 4 + 3 + 7 + 1 = 15, 7 + 3 + 0 = 10 => 1 + 5 = 6, 1 + 0 = 1 => 6 + 1 = 7
+        # It's cool that it's always 7. The proof is actually in the fact that we work with a decimal system but this is a neat little demonstration that it does work
+
+        # It smells. Suspiciously so
+        # Oh no, the kettle!!
+        raise Exception
```
Comment (jennasilla) [inline my_main.py:28]: Add text for 'exception'. Potentially write more tests for edge cases

File: my_main.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+def print_summm(n):
```
Comment (jennasilla) [inline my_main.py:30]: Rename function to print_sum

File: my_main.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        # 4 = 4, 3 + 7 + 1 + 7 + 3 + 0 = 21 =>
+        # 4 = 4, 2 + 1 = 3 =>
+        # 4 + 3 = 7
+        # 43|71730: 4 + 3 = 7, 7 + 1 + 7 + 3 + 0 = 18 => 7 = 7, 1 + 8 = 9 => 7 + 9 = 16 => 1 + 6 = 7
+        # 437|1730: 4 + 3 + 7 = 14, 1 + 7 + 3 + 0 = 11 => 1 + 4 = 5, 1 + 1 = 2 => 5 + 2 = 7
+        # 4371|730: 4 + 3 + 7 + 1 = 15, 7 + 3 + 0 = 10 => 1 + 5 = 6, 1 + 0 = 1 => 6 + 1 = 7
+        # It's cool that it's always 7. The proof is actually in the fact that we work with a decimal system but this is a neat little demonstration that it does work
+
+        # It smells. Suspiciously so
+        # Oh no, the kettle!!
```
Comment (jennasilla) [inline my_main.py:27]: Remove comments unrelated to the code

File: my_main.py [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    return FIRST_number, SECOND_number
+
+def split_and_print(numberone,numbertwo):
+    print("{} = {}, {} = {}".format(" + ".join(map(str, str(numberone))), print_summm(numberone),
+                                       " + ".join(map(str, str(numbertwo))),
+                                       print_summm(numbertwo)))
+    numberone = print_summm(numberone)
+    numbertwo = print_summm(numbertwo)
+    if len(str(numberone)) > 1 or len(str(numbertwo)) > 1:
+        #print(len(str(numberone)), len(str(numbertwo)))
```
Comment (jennasilla) [inline my_main.py:48]: Was this line used for testing? If so, please remove

File: my_main.py [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        split_and_print(numberone, numbertwo)
+    return print_summm(numberone), print_summm(numbertwo)
+
+def do_it_all(n):
+    n = str(n)
+    for i in range(len(n)-1):   # for 4371730 it would be 0,1,2,3,4,5
+        a_number, another_number_second_part = get_TWO_numbers_from_ONE_number(n,i)
+        print("{}|{}:".format(a_number, another_number_second_part))
+        #print(split_and_print(a_number, another_number_second_part))
```
Comment (jennasilla) [inline my_main.py:57]: Was this line used for testing? If so, please remove

File: README.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# code_review
+
+You ask what I want to do with it. I don't know yet, but I am sure I will know before I finish writing it.
```
Comment (jennasilla) [inline README.md:3]: Please update read-me file with relevant information

File: my_main.py [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+        # It smells. Suspiciously so
+        # Oh no, the kettle!!
+        raise Exception
+
+def print_summm(n):
+    s = str(n)
+    t = sum([int(d) for d in s])
+    return t
+
+def get_TWO_numbers_from_ONE_number(ALL_number,split_poInt):
```
Comment (jennasilla) [inline my_main.py:35]: Potentially change split_poInt to split_point unless uppercase "I" was intended

File: my_main.py [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+        # It's cool that it's always 7. The proof is actually in the fact that we work with a decimal system but this is a neat little demonstration that it does work
+
+        # It smells. Suspiciously so
+        # Oh no, the kettle!!
+        raise Exception
+
+def print_summm(n):
+    s = str(n)
+    t = sum([int(d) for d in s])
+    return t
```
Comment (jennasilla) [inline my_main.py:33]: Simplify function and potentially rename variables to something more meaningful

**Review by jennasilla — COMMENTED** [review]
Overall, the code runs as expected! We just need to clean things up a bit

File: my_main.py (hunk 4 above)
Comment (eshipunova) [reply my_main.py:28]: I would recommend giving an example here, so it's clear what you mean

File: my_main.py (hunk 5 above)
Comment (eshipunova) [reply my_main.py:30]: Yup I would probably refrain from giving a specific suggestion on what to rename to, but it is not wrong to mention

File: my_main.py (hunk 10 above)
Comment (eshipunova) [reply my_main.py:35]: Absolutely! Even if it was intended, we should stick to lower case here

File: my_main.py (hunk 11 above)
Comment (eshipunova) [reply my_main.py:33]: This is a great place to leave an example of what you mean by "simplify"

**Comment by eshipunova** [comment]
Nice work! Thanks for calling out testing, that was pretty important.

Here are a few things I would add next time:
* Test with different inputs and find out where it breaks
* Provide examples where your suggestion could be unclear
* Use "Suggested" and "Required" as a lead in to your comment, allowing the reviewee decide on as many things as it is reasonable
* Review file structure and suggest possible improvements to it
* Include some summary results in your CR summary, it's nice to see at a glance the results of the code review
